environment.yml

```diff
@@ -20,5 +21,7 @@ dependencies:
  - pyyaml
  - lxml
  - scipy
+ - ruamel_yaml
+ - pyvisa=1.8
```

Comment:
pyvisa was from pip because Jens told me that conda has weird versions (don't remember exactly the problem). Has it been fixed?